DQM/L1TMonitor/src/SealModule.cc

```diff
@@ -27,6 +27,9 @@ DEFINE_FWK_MODULE(L1TRPCTF);
 #include <DQM/L1TMonitor/interface/L1TGMT.h>
 DEFINE_FWK_MODULE(L1TGMT);
 
+#include <DQM/L1TMonitor/interface/L1MUTK.h>
+DEFINE_FWK_MODULE(L1MUTK);
```

Comment:
as far as I know, this macro can actually be added just in the L1TkMuonMonitor.cc file